pkg/broker/ingress/multi_topic_decouple_sink_test.go

```diff
@@ -54,7 +57,7 @@ func TestMultiTopicDecoupleSink(t *testing.T) {
 			name: "happy path single broker",
 			brokerConfig: &config.TargetsConfig{
 				Brokers: map[string]*config.Broker{
-					"test_ns_1/test_broker_1": {DecoupleQueue: &config.Queue{Topic: "test_topic_1", State: config.State_READY}},
+					"test_ns_1/test_broker_1": {DecoupleQueue: &config.Queue{Topic: "test_topic_1", State: config.State_READY}, Targets: brokerTargets},
```

Comment:
The `Targets` and `DecoupleQueue` fields can be in their own new line.

Reply:
I don't think line length is the concern here, in our code base, you do normally have multiple line for struct initialization. That's why `Brokers:` is in its own line.